### DISCUSSION TITLE
Installer triggers Windows SmartScreen block

### DISCUSSION THREAD

**Discussion by yllekz** [Ruben2776/PicView#39] [General]
**Describe the bug**
PicView's installer gets blocked by Windows SmartScreen.

**To Reproduce**
1. Run the installer
2. See the SmartScreen page

**Expected behavior**
The installer should be properly formatted so it does not get stopped by SmartScreen.

**Screenshots**
![image](https://user-images.githubusercontent.com/28446638/133502780-48a02d41-5243-465f-a4d0-d3b1725760af.png)

**Additional context**
Windows 10 21H1 19043.1165

**Comment by Ruben2776** [comment]
Hi there, thanks for getting in touch.

This isn't something I can fix my own without paying hundreds of dollars for a a code signing certificate...
The application is signed as you can see by right clicking `PicView-Setup.exe` in Explorer and selecting `Properties` and then `Digital Signatures`. 

You can read more here https://docs.microsoft.com/en-us/archive/blogs/ie/microsoft-smartscreen-extended-validation-ev-code-signing-certificates or search for code signing certificate if you're curious, but basically it will go after after enough people have downloaded and installed it.
